### ISSUE TITLE
is there a way to use reserved keywords as a message field?

### ISSUE THREAD

**Issue by iurii2002** [ApeWorX/eip712#33]
For example, I want to create a message of the following structure

class Mail(EIP712Message):
    _name_ = "Ether Mail"
    _version_ = "1"

    from: "address"
    receiver: "address"

**Comment by fubuloubu** [comment]
Not currently.

A way to rename members for encoding purposes is needed, hence why I wanted to pursue #22 as a solution, which would allow this (among other things)

**Comment by unparalleled-js** [comment]
Yes, if we do #22, we can use field aliases, would that work @fubuloubu ?
So this is blocked by #22
